### PR TITLE
[FLEX-533] changes for adding classname to CardBlock

### PR DESCRIPTION
**Pull request by psivakumaran**

### CODE REVIEW

File: src/cards/Card.tsx [hunk 1]
```diff
@@ -16,8 +16,11 @@ export const CardImage: React.FunctionComponent = ({children}) => {
     return <div className={ClassNames.CARD_IMAGE}>{children}</div>;
 };
 
-export const CardBlock: React.FunctionComponent = ({children}) => {
-    return <div className={ClassNames.CARD_BLOCK}>{children}</div>;
+export type CardBlockProps = {
```
Comment (akate1) [inline src/cards/Card.tsx]: You don't need to export CardBlockProps. As currently you are using them in same file only.

Comment (ben-schumacher) [reply src/cards/Card.tsx]: I agree here.

Comment (psivakumaran) [reply src/cards/Card.tsx]: Done.

**Comment by psivakumaran** [comment]
> @psivakumaran I think the changes look fine, but just wondering why you need to set custom className on CardBlock? Can you provide some context please?

I have added these to give a grey background to the Objectstore name and menu section.
When I directly use div/span, there is a white border around the div which is part of the card block

**Pull request merged**